src/Raven.Server/Documents/ETL/Providers/SQL/RelationalWriters/SqlConnectionStringUtil.cs

```diff
@@ -0,0 +1,14 @@
+namespace Raven.Server.Documents.ETL.Providers.SQL.RelationalWriters;
+
+public static class SqlConnectionStringUtil
+{
+    public static string GetConnectionStringWithOptionalEncrypt(string connectionString)
+    {
+        if (connectionString.EndsWith(';') == false)
```

Comment:
maybe add 
`connectionString = connectionString.Trim();` in first line of this method?

Reply:
Added